### PR TITLE
list_of_events: Remove mail_cache_lookup_finished event

### PR DESCRIPTION
**Pull request by sirainen**
DOV-4946

### CODE REVIEW

File: source/admin_manual/list_of_events.rst [hunk 1]
```diff
@@ -1290,6 +1290,8 @@ mail_cache_lookup_finished
 
 .. versionadded:: 2.3.15
 
+.. versionremoved:: 2.3.18 Removed for performance reasons.
```
Comment (cmouse) [inline source/admin_manual/list_of_events.rst:1293]: Shouldn't the text go indented next line?

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -2832,18 +2834,24 @@ in to the backend.
...
-| Field           | Description                                          |
-+=================+======================================================+
-| source_port     | Source port where proxy connection originated from   |
-+-----------------+------------------------------------------------------+
-| error           | If login to destination failed, contains the error.  |
-+-----------------+------------------------------------------------------+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| source_port         | Source port where proxy connection originated from   |
```
Comment (cmouse) [inline source/admin_manual/list_of_events.rst:2848]: dot missing.

File: source/admin_manual/list_of_events.rst (hunk 1 above)
Comment (sirainen) [reply source/admin_manual/list_of_events.rst:1293]: The end result looks now like this:

`Removed in version 2.3.18: Removed for performance reasons.`

So I think it's ok?

File: source/admin_manual/list_of_events.rst (hunk 2 above)
Comment (sirainen) [reply source/admin_manual/list_of_events.rst]: This commit wasn't supposed to be here.. But will fix it in https://github.com/dovecot/documentation/pull/351

File: source/admin_manual/list_of_events.rst (hunk 1 above)
Comment (cmouse) [reply source/admin_manual/list_of_events.rst:1293]: ok

**Review by cmouse — APPROVED** [review]

**Pull request merged**